src/adapters/coffeeScriptAdapter.ts

```diff
@@ -11,12 +11,20 @@ export class CoffeeScriptAdapter implements TSAdapter {
 
   public getFileContent(): string {
     const coffeeScriptContent = this.document.getText();
-    const options = { disableSuggestionComment: true };
+    const options = getOptionsFromConfig();
     return convert(coffeeScriptContent, options).code;
   }
 
   public getFileName(): string {
     const originalFileName = this.document.fileName;
     return originalFileName.replace(/\.coffee/, '.ts');
   }
-}
\ No newline at end of file
+}
+
+export function getOptionsFromConfig() {
```

Comment:
Could we move this helper into [configUtils.ts](https://github.com/alecclarke/to-typescript/blob/master/src/configUtils.ts), similar to `getEditorConfig`?

Also, it might be worth adding to the existing `configUtils` unit tests to ensure the default decaf config is returned.

Reply:
Sounds good, just updated this.

The only issue with the test is that, I can't find a good way to ignore global user settings.
If coffeescript options are specified in there, the tests fail.

Reply:
Oh that's a good catch! I believe there's a `launch` [setting](https://code.visualstudio.com/api/working-with-extensions/testing-extension#the-test-script) where the workspace used in the tests can be specified, but I'll have to do a bit of digging to figure that out.

For now, if we add these tests in and use CI to verify the pass as expected in a default workspace, I think that's good for now. I can follow up with the test specific workspace configuration after.